src/pages/[platform]/build-a-backend/data/data-modeling/add-fields/index.mdx

```diff
@@ -87,6 +87,8 @@ a.schema({
 });
 ```
 
+<InlineFilter filters={["javascript", "angular", "react-native", "react", "nextjs", "vue", "android", "flutter"]}>
```

Comment:
This PR continues to include android and flutter code snippets under JS snippets, as this is only scoped to  address the Swift ones for now.